### DISCUSSION TITLE
Next.js setup documentation outdated

### DISCUSSION THREAD

**Discussion by tedishero** [microsoft/fluentui#28676] [Q&A] [answered]
Hi,
I recently checked out the next.js integration [documentation ](https://react.fluentui.dev/?path=/docs/concepts-developer-server-side-rendering-next-js-setup--page) in fluent V9 and the _document.tsx and _app.tsx described in the documentation doesn't exist in NextJs's new AppRouter format. Can we update the documentation for AppRouter format as well?

**Comment by layershifter** [comment, marked as answer]
Support for appDir is tracked in #28196.
